src/detect-oxc.ts

```
@@ -1,9 +1,35 @@
+import type MagicString from 'magic-string'
 import type { Program } from 'estree'
```

ESLint

[error] Expected "estree" to come before "magic-string".

(perfectionist/sort-imports)